sdk/core/core/inc/az_xml_value.h

```diff
@@ -0,0 +1 @@
+#pragma once
```

Comment:
Don't use `#pragma once` in your real implementation

Reply:
Cool, removed :)